### ISSUE TITLE
Mixed up descriptions after hydration of docs generated by the CLI

### ISSUE THREAD

**Issue by aryehb** [Redocly/redoc#1121]
I'm having an issue with the CLI, where when the schema get re-hydrated, the field details get mixed up.

When I use the CLI to generate documentation with the schema below, the initial generated docs are fine. However, after they are re-hydrated, the descriptions will be mixed up, and I'll see `Field 6` as the description for `field1`. This only happens when there are at least 11 fields, and when using the `--options.requiredPropsFirst` flag, regardless of whether there are required fields in the schema.

```yaml
openapi: 3.0.0
info:
  title: API
  version: 1.0.0
paths:
  /:
    post:
      requestBody:
        required: true
        content:
          application/json:
            schema:
              type: object
              properties:
                field1:
                  type: string
                  description: Field 1.
                field2:
                  type: string
                  description: Field 2.
                field3:
                  type: string
                  description: Field 3.
                field4:
                  type: string
                  description: Field 4.
                field5:
                  type: string
                  description: Field 5.
                field6:
                  type: string
                  description: Field 6.
                field7:
                  type: string
                  description: Field 7.
                field8:
                  type: string
                  description: Field 8.
                field9:
                  type: string
                  description: Field 9.
                field10:
                  type: string
                  description: Field 10.
                field11:
                  type: string
                  description: Field 11.
```

Sample screenshot:

![image](https://user-images.githubusercontent.com/49538279/69981980-657c0a80-1501-11ea-840b-d7e0e24597c9.png)

**Comment by fabong** [comment]
We are having the same issue. Also, it seems that this issue could be linked: https://github.com/Redocly/redoc/issues/1061.

**Comment by aryehb** [comment]
Good point. I didn't see that issue.

**Comment by fabong** [comment]
I just tested this PR: https://github.com/Redocly/redoc/pull/1107 and it fixes our issue.

**Comment by yannan** [comment]
I have the same issue. So I pointed this bug in #1104 and try to fix it in #1107
